### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by samwgoldman**
My bad :neutral_face:

### CODE REVIEW

File: docs/01-dynamic-type-tests.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ function bar(x: ?string): string {
 }
 {% endhighlight %}
 
-Read more about <a href="docs/nullable-types.html">Maybe Types</a>.
+Read more about <a href="/docs/nullable-types.html">Maybe Types</a>.
```
Comment (mroch) [inline docs/01-dynamic-type-tests.md]: let's do `../docs/nullable-types.html` so that it also works on facebook.github.io/flow/

Comment (samwgoldman) [inline docs/01-dynamic-type-tests.md]: That site is pretty broken right now. There are a bunch of other absolute hrefs on this site already. Can we merge this as-is and I'll got back in and fix all the hrefs in a separate PR?

Comment (mroch) [inline docs/01-dynamic-type-tests.md]: (this'll matter more after #584)

**Comment by mroch** [comment]
yup, i'll just change #584 to include that too

**Pull request merged**